langchain/src/vectorstores/qdrant.ts

```diff
@@ -115,6 +125,8 @@ export class QdrantVectorStore extends VectorStore {
       payload: {
         content: documents[idx].pageContent,
         metadata: documents[idx].metadata,
+        ...(documents[idx].customPayload ? documents[idx].customPayload != customPayload ? 
+          documents[idx].customPayload : customPayload : customPayload),
```

Comment:
Could you just do something like this?
```suggestion
          documents[idx].customPayload ?? customPayload),
```
Not sure if this is valid though